stdlib/public/runtime/Metadata.cpp

```diff
@@ -142,65 +226,148 @@ static GenericMetadataCache &unsafeGetInitializedCache(
   return lazyCache->unsafeGetAlreadyInitialized();
 }
 
+#ifndef NDEBUG
```

Comment:
It looks like this conditional doesn't match the uses of `_objc_empty_cache` (or vice versa). https://ci.swift.org/job/oss-swift_tools-RA_stdlib-RD_test-simulator/714 rdar://problem/38132762

Reply:
Yes, that was noticed yesterday and fixed in #14970.